Mention online help for SolidFire volume driver

scaleoutsean opened the pull request:
It seems useful to have it mentioned in readme.md

scaleoutsean committed:
Mention online help for SolidFire volume driver

It seems useful to have them mentioned in readme.md

scaleoutsean committed:
Added SolidFire driver CLI example

.. and grouped the two commands together at the end of Usage Examples

j-griffith commented:
These are good adds Sean, and I agree pointing out that there's a CLI component is something we should def have here.  Could you remove the prefix though (/usr/local), the install script writes directly to /usr/bin.. and if they're running from source (or the installed manually) the binary could be anywhere.

scaleoutsean committed:
Removed custom binary path, reorg

- /usr/local/bin=>/usr/bin
- Reodered and enhanced some parts to reflect the improvements made in v1.3.1
- Changed the Copyright line to be in line with other NetApp projects (used https://github.com/NetApp/santricity-webapi-javasdk/edit/master/LICENSE.md as template)

scaleoutsean commented:
Point taken. I think I fixed that and I added some stuff to document the improvements in v1.3.1.
Hopefully I didn't make new mistakes!
